tide/core/scene/ContentWindow.h

```diff
@@ -144,7 +144,7 @@ class ContentWindow : public Rectangle
     /** @return the unique identifier for this window. */
     const QUuid& getID() const;
 
-    /** Is the window a panel */
+    /** Is the window a panel. */
```

Comment:
Done for the the first part.
I'm not surprised that you don't know because it has been an evolving and never clearly defined concept. It was introduced as a generalisation intended to replace the special handling applied to the "Dock" window. The WindowType enum gives the following description:
PANEL       // A panel window - always interative, cannot be focused
I'm going to update it to:
PANEL       // an overlay window without a control bar, cannot be focused or fullscreen